particles/List/Items.arcs

```diff
@@ -20,41 +12,14 @@ particle ItemMultiplexer in 'source/Multiplexer.js'
   host HostedItemInterface hostedParticle
   consume set of item
 
-// TODO(sjmiles): recipe is the minimum coalescable artifact, which is good because we need to be able specify
-// handle fates before colascing ... is there a way to combine the declarations when the recipe has only one Particle?
-//recipe ItemMultiplexer
-  // TODO(sjmiles): restricting fate
-  // TODO(sjmiles): without `#items` this recipe doesn't coalese, why?
-//  use #items as list
-//  ItemMultiplexer
-//    list = list
-
-//particle List in 'source/List.js'
-//  in [~anyType] list
-//  consume root #items
-//    must provide set of item
-//      handle items
-//  description `show ${list}`
-
-//recipe List
-//  use #items as items
-//  List
-//    items = items
-//  ItemMultiplexer
-//    list = items
-
 particle Items in 'source/Items.js'
   in [~anyType] list
   consume root #items
     provide preamble
-    must provide set of item
-      handle list
+    provide set of item
```

Comment:
just curios:
- is there a problem with `must provide`?
- does the handle restriction for provided slots no longer work?

Reply:
No problem necessarily, but generally recipe are brittle and hard to get working. I wanted to simplify the declarations.